### ISSUE TITLE
Why lerna does not bump the first major version when using conventional commits and major version is zero?

### ISSUE THREAD

**Issue by lusatiro** [lerna/lerna#2766]
## Expected Behavior

Bump 0.0.1 -> 1.0.0 when making a BREAKING CHANGE commit.

## Current Behavior

Does not bump the major version. Only minor and patch.

I tried find in the docs, but i didn't find nothing tha explain the behaviour. I cannot make manually the first bump of major cause I'm using Github Actions for handle this.

**Comment by sempasha** [comment]
From [semver 2 spec](https://semver.org/spec/v2.0.0.html):

>  4 . Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.
>  5. Version 1.0.0 defines the public API. The way in which the version number is incremented after this release is dependent on this public API and how it changes.
>  8. Major version X (X.y.z | X > 0) MUST be incremented if any backwards incompatible changes are introduced to the public API. ...

When package has a version of `0.0.1` it seems to be in active development, and has no stable API, any new version may have a breaking changes.

Major version must incremented on breaking change only after package version `1.0.0` has been released.

**Comment by matzeeable** [comment]
I think this is a duplicate of #2761

I have prepared a `.patch` for this change but had not yet time to create a PR for this.

**Comment by william-will-angi** [comment]
From the code: https://github.com/lerna/lerna/blob/main/core/conventional-commits/lib/recommend-version.js#L68
```
            // According to semver, major version zero (0.y.z) is for initial
            // development. Anything MAY change at any time. The public API
            // SHOULD NOT be considered stable. The version 1.0.0 defines
            // the (initial stable) public API.
            //
            // To allow monorepos to use major version zero meaningfully,
            // the transition from 0.x to 1.x must be explicitly requested
            // by the user. Breaking changes MUST NOT automatically bump
            // the major version from 0.x to 1.x.
            //
            // The usual convention is to use semver-patch bumps for bugfix
            // releases and semver-minor for everything else, including
            // breaking changes. This matches the behavior of `^` operator
            // as implemented by `npm`.
```

**Comment by JamesHenry** [comment]
Closing as this is either a misunderstanding of the semver spec, or a feature request duplicate of https://github.com/lerna/lerna/issues/2761
